content/docs/command-reference/freeze.md

```diff
@@ -1,32 +1,32 @@
-# lock
+# freeze
 
-Lock a [DVC-file](/doc/user-guide/dvc-file-format)
-([stage](/doc/command-reference/run)). Use `dvc unlock` to unlock the file.
+Freeze a [stage](/doc/command-reference/run). Use `dvc unfreeze` to unfreeze the
+stage.
 
 ## Synopsis
 
 ```usage
-usage: dvc lock [-h] [-q | -v] targets [targets ...]
+usage: dvc freeze [-h] [-q | -v] targets [targets ...]
 
 positional arguments:
-  targets        DVC-files to lock.
+  targets        stages to freeze.
 ```
 
 ## Description
 
-`dvc lock` causes any DVC-file to be considered _not changed_ by `dvc status`
-and `dvc repro`. Stage reproduction will not execute regenerate
-<abbr>outputs</abbr> of locked stages, even if some dependencies have changed,
+`dvc freeze` causes any stage to be considered _not changed_ by `dvc status` and
+`dvc repro`. Stage reproduction will not execute the command or regenerate
+<abbr>outputs</abbr> of frizen stages, even if some dependencies have changed,
 and even if `--force` is provided.
```

Comment:
Thanks for fix! I think here we meant to simplify to just
```suggestion
`dvc repro`. Stage reproduction will not regenerate <abbr>outputs</abbr>
of frozen stages, even if some dependencies have changed, and even if
`--force` is provided.
```
Also, typo fixed ^ 😬